src/Session.php

```diff
@@ -363,6 +362,7 @@ public function wait($time, $condition = 'false')
      */
     public function resizeWindow($width, $height, $name = null)
     {
+        $this->start();
```

Comment:
But @peterrehm is using them and they work perfectly. Otherwise there is no way to open window initially with given size so that upon page visiting we won't be resizing it again.